addon/mixins/node-actions.js

```diff
@@ -144,7 +142,14 @@ export default Ember.Mixin.create({
             });
             // After user has been saved, add user as a contributor
             return user.save().then(user => {
-                this.send('addContributor', user.id, permission, isBibliographic);
+                var node = this.get('_node');
```

Comment:
I'm a little concerned about the duplicate code here, but we can dig into that later once other higher priorities have been cleared up.